src/renameObjectKey.ts

```diff
@@ -1,6 +1,6 @@
 // renames a key while trying to preserve key ordering
 export const renameObjectKey = (obj: object, oldKey: string, newKey: string) => {
-  if (!obj || !obj[oldKey] || oldKey === newKey) {
+  if (!obj || !obj.hasOwnProperty(oldKey) || oldKey === newKey) {
```

Comment:
```suggestion
  if (!obj || Object.hasOwnProperty.call(obj, oldKey) || oldKey === newKey) {
```
Suggest the above, just in case such an object with a non-standrd `hasOwnProperty` is given.

Reply:
👍 but `if (!obj || !Object.hasOwnProperty.call(obj, oldKey) || oldKey === newKey) {` ;)